types/intl-tel-input/index.d.ts

```diff
@@ -156,6 +156,12 @@ declare namespace intlTelInput {
          */
         customPlaceholder?: ((selectedCountryPlaceholder: string, selectedCountryData: CountryData) => string) | undefined;
 
+        /**
+         * Additional classes to add to the parent div..
+         * Default = ''
```

Comment:
```suggestion
         * @default ''
```

Reply:
@peterblazejewicz I matched the format from the others descriptions, would you like me to also update them or just this one ?

Reply:
we adding JSDoc for users, not to match readme. There is no source code JSDoc, so, your call, sir.

Reply:
I meant in this file, every other variables uses the `Default = value``` format :)